server/src/routes/clientRouter.js

```diff
@@ -3,7 +3,13 @@ import getClientIndexPath from "../config/getClientIndexPath.js";
 
 const router = new express.Router();
 
-const clientRoutes = ["/", "/user-sessions/new", "/users/new"];
+const clientRoutes = [
+  "/",
+  "/user/profile",
+  "/user-sessions/new",
+  "/user-sessions/current",
```

Comment:
In terms of scoping PRs, I would hold off on adding these additional routes until you actually build these pages ("/user/profile" and "/user-sessions/current")